frontend/src/views/WheresMyGene/components/CellInfoSideBar/style.ts

```diff
@@ -127,3 +127,8 @@ export const NoMarkerGenesDescription = styled("span")`
     `;
   }}
 `;
+
+export const InfoButtonWrapper = styled("span")`
```

Comment:
this seems to be the same as `frontend/src/views/WheresMyGene/components/HeatMap/components/YAxisChart/style.ts:79`, maybe worth extracting out as common component in `frontend/src/views/WheresMyGene/components/common/style.ts` or something?